riff-cli/cmd/publish.go

```diff
@@ -61,6 +61,7 @@ will post '{"hello":"world"}' as json to the 'concat' topic and wait for a reply
 
 `,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
```

Comment:
Do we always want to silence usage, or only when we have a known http error?

Reply:
As a matter of convention, we should often silence usage as soon as we enter `runE`.
Misconfiguration errors (that should then show usage) are normally handled in `preRunE`, but when we reach `runE` I think it's safe to assume we understood what the user wanted